server/src/com/sixsq/slipstream/ssclj/resources/spec/device_dynamic.cljc

```diff
@@ -47,25 +47,33 @@
 (s/def :cimi.device-dynamic/wifiAddress ::cimi-core/nonblank-string)
 (s/def :cimi.device-dynamic/ethernetThroughputInfo (s/coll-of string?))
 (s/def :cimi.device-dynamic/wifiThroughputInfo (s/coll-of string?))
+(s/def :cimi.device-dynamic/sensorType (s/coll-of string?))
+(s/def :cimi.device-dynamic/sensorModel ::cimi-core/nonblank-string)
```

Comment:
This should also be a list of strings. But it may be better to only have one list of JSON sensor descriptors, otherwise you'd need to ensure all three lists are of the same size and in the same order.

@cjdcordeiro, how flexible are the types here? Can this be defined as something resembling

 ```
listof(
    object(
        type=listof(string),
        model=string,
        connection=mapof(string -> string)
    )
)
```

or is it cleaner to just use a list of strings and agree that those strings will be in a specific format?

Reply:
@sstanovnik are talking about the `sensorConnection` ? Is it really essential to change the datatype for `sensorConnection` or if I'll provide you with a string of the `sensorConnection` information is it okay for you?

Reply:
I am _also_ talking about `sensorConnection`, as both need to be collections, an element for each sensor. That's why I think it might be more appropriate to only have one element instead of three.

Reply:
okay @sstanovnik just to confirm one thing are you expecting the `sensorConnection` value like as - `"{sensorConnection": [{'baudRate': 5600, 'gpioPin': 23}]}`

Reply:
Yes, in the case with three top-level endpoints, exactly that.